step.yml

```diff
@@ -181,14 +181,25 @@ inputs:
         - "up_until_maximum_repetitions"
   - maximum_test_repetitions: 3
     opts:
-      title: Maximum Test Repetitions
+      title: Maximum Test Repetitions (Available in Xcode 13+)
       category: Test Repetition
       summary: The maximum number of times a test will repeat based on Test Repetition Mode.
       description: |-
         The maximum number of times a test will repeat based on Test Repetition Mode.
         
-        Should be more than 1 if Test Repetition Mode (`test_repetition_mode`) is other than `none`.
+        Should be more than 1 if the Test Repetition Mode (`test_repetition_mode`) is other than `none`.
       is_required: true
+  - relaunch_tests_for_each_repetition: "no"
+    opts:
+      title: Relaunch Tests for Each Repetition (Available in Xcode 13+)
+      summary: If enabled, tests will launch in a new process for each repetition.
+      description: |-
+        If enabled, tests will launch in a new process for each repetition.
+
+        By default, tests launch in the same process for each repetition.
```

Comment:
Can we elaborate what it means to launch in the same process? Why it's useful in practice, what problem it solves, etc. I would also link the relevant documentation from Apple on the `xcodebuild` flag (if there's any)

Reply:
Thanks for raising this!

`xcodebuild`'s command line documentation is not more verbose than our description; in fact, I almost copied it word-to-word:
```
$ xcodebuild -h
...
-test-repetition-relaunch-enabled YES|NO                 Enable or disable, tests repeating in a new process for each repetition.  Must be used in conjunction with -test-iterations, -retry-tests-on-failure, or -run-tests-until-failure. If not specified, tests will repeat in the same process.
```
`man xcodebuild` mentions [Xcode Builds Settings Reference](https://help.apple.com/xcode/mac/current/#/itcaec37c2a6), but it doesn't say anything about Test Repetition.

Would it make sense to talk more verbosely about this and its use case in DevCenter only?